homeassistant/components/sensor/google_home_alarm.py

```diff
@@ -0,0 +1,103 @@
+"""
+Sensor for retrieving latest alarm from Google Home.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.google_home_alarm/
+"""
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_HOST, CONF_MONITORED_CONDITIONS)
+import homeassistant.util.dt as dt_util
+
+REQUIREMENTS = ['ghlocalapi==0.1.0']
+
+ICON = 'mdi:alarm'
+
+SENSOR_TYPE_TIMER = 'timer'
+SENSOR_TYPE_ALARM = 'alarm'
+
+SENSOR_TYPES = {
+    SENSOR_TYPE_TIMER: "Timer",
+    SENSOR_TYPE_ALARM: "Alarm",
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=['alarm', 'timer']):
```

Comment:
There are 2 monitored conditions.... is it really needed to make a configuration option out of this?

Also with this in mind: home-assistant/architecture#100

I would like to opt to remove this option and just expose both.

Reply:
Something like my latest commit?